_data/exhibitions.yml

```diff
@@ -1,17 +1,23 @@
-- title: dotCSS
-  link: https://www.dotcss.io/
-  date: 4. Dec 2019
+- title: ePages Hackathon
+  link:
```

Comment:
I was thinking about adding a link to a blog post here (https://developer.epages.com/blog/events/epages-hackathon-2017/) . This way people who are interested but do not know what this is about might get an idea?

What do you think? Is it to old?